### ISSUE TITLE
publish package in packgist

### ISSUE THREAD

**Issue by Grummfy** [intellitrend/zabbixapi-php#4]

**Comment by claytonfinney** [comment]
+1! Please :)

**Comment by intellitrend-team** [comment]
Good idea. Need to look into this.

**Comment by claytonfinney** [comment]
Any update on this? Would love to use this library with composer :)

**Comment by intellitrend-team** [comment]
We updated the library to be compatible with Zabbix version 5.4 and also added **composer support** and published on Packagist, see here [intellitrend/zabbixapi](https://packagist.org/packages/intellitrend/zabbixapi).

There is also a new working example using composer in `examples/composer`.

**Comment by daften** [comment]
Pinning on dev-master is not really best practice, so can this library also make sure the tagged versions from here are published on packagist automatically? I'm not sure why this is not happening, normally tags with semver (including with v prefix) should be picked up automatically, but it's not happening here for some reason.

Sidenote: the changelog is really helpful, but my personal preference is a separate CHANGELOG.md file, to not clutter the helpful README file itself. Is that something that could be considered?

Thanks!

**Comment by intellitrend-team** [comment]
The version tags were in a format that Packagist didn't recognize due to an additional dot after the "v". We have created new tags for v3.0.2 and v3.1.0 and they now appear in the version list. The examples also have been updated to use the latest stable version instead of dev-master.

Also, good point about the changelog file! As the list grows, it should definitely be moved to a separate file, yes.
